.github/workflows/script_test.yml

```diff
@@ -1,34 +0,0 @@
-name: Script Test
```

Comment:
why remove this file? it is used to test some script

Reply:
The file runs network test on master and release 1.0, while the job `integration_test` covers the same test on master.  Since we are no longer maintaining 1.0, it becomes useless.

Reply:
In fact, this file is used to test `install.sh`,  make sure this script can install Chaos Mesh success

Reply:
oops, in that case that seems necessary. I'll revert this.